tests/static/all_question_types.tsv

```diff
@@ -28,6 +28,6 @@ Question	Type	Text (en)	Text (hin)	Audio (en)	Audio (hin)	Image (en)	Image (hin)
 /question22/question23/question28	Password	question28	question28									no
 /question22/question23/question29	Signature Capture	question29	question29									no
 /question22/question23/question7	Android App Callout	question7	question7									no
-	Base											no
 /question20	Hidden Value											no
-/question32	Hidden Value										1 + 2	no
\ No newline at end of file
+/question32	Hidden Value										1 + 2	no
+	Base											no
```

Comment:
I changed this because it seemed to just fall out of the old implementation and I didn't see any comments about how we want this to stay the same.

Basically if something is only in the control path and at the end of the control body (like this piece of custom xml) it will end up getting put at the end.

Is this acceptable? I could only find this one edge case and it didn't seem to affect the end product (didn't change the xml on this same test)